protocol/code_review.md

```diff
@@ -0,0 +1,75 @@
+* Avoid using terms that could be seen as referring to personal traits. ("dumb",
+  "stupid"). Assume everyone is attractive, intelligent, and well-meaning.
+* Be explicit. Remember people don't always understand your intentions online.
+* Be humble. ("I'm not sure - let's look it up.")
+* Don't use hyperbole. ("always", "never", "endlessly", "nothing")
+* Don't use sarcasm.
+* Keep it real. If emoji, animated gifs, or humor aren't you, don't force them.
+  If they are, use them with aplomb.
+* Talk in person if there are too many "I didn't understand" or "Alternative
+  solution:" comments. Post a follow-up comment summarizing offline discussion.
+
+Having Your Code Reviewed
+-------------------------
+
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* Don't take it personally. The review is of the code, not you.
+* Explain why the code exists. ("It's like that because of these reasons. Would
+  it be more clear if I rename this class/file/method/variable?")
+* Extract some changes and refactorings into future tickets/stories.
+* Link to the code review from the JIRA ticket/story. ("Ready for review:
+  https://github.com/organization/project/pull/1")
+* Push commits based on earlier rounds of feedback as isolated commits to the
+  branch. Do not squash until the branch is ready to merge. Reviewers should be
+  able to read individual updates based on their earlier feedback.
+* Seek to understand the reviewer's perspective.
+* Try to respond to every comment.
+* Wait to merge the branch until Bamboo plan tells you the test suite is green in the branch.
+* Merge once you feel confident in the code and its impact on the project.
+
+Reviewing Code
+--------------
+
+Understand why the code is necessary (bug, user experience, refactoring). Then:
+
+* Communicate which ideas you feel strongly about and those you don't.
+* Identify ways to simplify the code while still solving the problem.
+* If discussions turn too philosophical or academic, move the discussion offline
+  to a regular Friday afternoon technique discussion. In the meantime, let the
+  author make the final decision on alternative implementations.
+* Offer alternative implementations, but assume the author already considered
+  them. ("What do you think about using a custom validator here?")
+* Seek to understand the author's perspective.
+* Sign off on the pull request with a :thumbsup: or "Ready to merge" comment.
+
+Style Comments
+--------------
+
+Reviewers should comment on missed [style](https://github.com/bbatsov/ruby-style-guide)
+guidelines. Example comment:
+
+    [Style](https://github.com/bbatsov/ruby-style-guide)
+
+    > Order resourceful routes alphabetically by name.
+
+An example response to style comments:
+
+    Whoops. Good catch, thanks. Fixed in a4994ec.
+
+If you disagree with a guideline, open an issue on the guides repo rather than
+debating it within the code review. In the meantime, apply the guideline.
```

Comment:
Maybe the comment I added [above](https://github.com/ucsdlib/DeveloperGuides/pull/1/files#r38467345) belongs here in a new section called "Scope"?  Keeping code review tightly scoped is very important to keeping things moving.